extensions/notification/utils/message.js

```diff
@@ -0,0 +1,79 @@
+const { versioning } = require('arsenal');
+const versionIdUtils = versioning.VersionID;
+
+const constants = require('../constants');
+
+/**
+ * Add attributes to an internal notification message,
+ * with its log entry counterpart.
+ * @param  {Object} logEntry - log entry that generated the message
+ * @param  {Object} message - initial message generated
+ * @return {Object} updated message object
+ */
+function addLogAttributes(logEntry, message) {
+    const { eventMessageProperty } = constants;
+    const msg = {};
+    for (const [key, value] of Object.entries(eventMessageProperty)) {
+        let val = logEntry[value];
+        if (typeof val === 'number') {
+            val = val.toString();
```

Comment:
Is this required by the spec (convert numbers to strings)?

Reply:
yes everything appears to be string, https://docs.aws.amazon.com/AmazonS3/latest/dev/notification-content-structure.html